sycl/doc/EnvironmentVariables.md

```diff
@@ -245,6 +245,21 @@ Supported tracing levels are in the table below
 | 2 | Enable tracing of the UR calls |
 | -1 | Enable all levels of tracing |
 
+### `SYCL_CACHE_TRACE` Options
+
+`SYCL_CACHE_TRACE` enables tracing of different SYCL caches and kernel_compiler. It accepts one of the values from the table below:
+
+| Option | Description |
+| ------ | ----------- |
```

Comment:
to me, this seems weird to list out like this.  I think it would be better to explain to the reader that the value is a bit mask,  0x01 for disk, 0x02 for memory, 0x04 for kernel_compiler. 

I'm I'm not mistaken, we have at least one other environment var which is a bitmask ( L0 USM RESIDENT?)